### PR TITLE
Team Judging Registration

### PR DESCRIPTION
**Pull request by SpencerKaiser**
## Pre-Requisites
- [x] Yes, I updated [Authors.md](../Authors.md) **OR** this is not my first contribution
- [x] Yes, I included and/or modified tests to cover relevent code **OR** my change is non-technical
- [x] Yes, I wrote this code entirely myself **OR** I properly attributed these changes in [Third Party Notices](../THIRD-PARTY-NOTICES.txt)

## Description of Changes

Implementing the Slack UI functionality to allow teams to request to be judged by the sponsorship team.

### CODE REVIEW

**Comment by SpencerKaiser** [comment]
@johnkahn I recreated this as a draft as it's currently a WIP. Let me know if you have any feedback on the new stuff!

**Comment by SpencerKaiser** [comment]
@johnkahn this is ready for review! 🌮

File: src/entities/team.ts [hunk 1]
```diff
@@ -17,11 +18,11 @@ export class Team extends BaseEntity {
   @Column()
   name: string;
 
-  @Column()
-  location: string;
+  @Column({ unique: true })
+  tableNumber: string;
```
Comment (johnkahn) [inline src/entities/team.ts]: Should this be a number?

Comment (SpencerKaiser) [reply src/entities/team.ts]: I check if it's a number when I collect it, but I thought maybe a number might be less brittle... thoughts?

Comment (johnkahn) [reply src/entities/team.ts]: Well if we want to make sure it's a number, I would prefer that we save it as a number in the database. Easier to migrate `number -> string` than `string -> number`

File: src/slack/actions/registerTeam.ts [hunk 2]
```diff
@@ -64,9 +64,9 @@ function register(bolt: App): void {
 
     const teamName = retrieveViewValuesForField(view, registerTeamViewConstants.fields.teamName, 'plainTextInput') as string;
     const projectDescription = retrieveViewValuesForField(view, registerTeamViewConstants.fields.projectDescription, 'plainTextInput') as string;
-    const tableNumber = retrieveViewValuesForField(view, registerTeamViewConstants.fields.tableNumber, 'plainTextInput') as string;
+    const tableNumber = +retrieveViewValuesForField(view, registerTeamViewConstants.fields.tableNumber, 'plainTextInput');
```
Comment (johnkahn) [inline src/slack/actions/registerTeam.ts]: Can we use `parseInt()` here? The plus is confusing to those that don't know how Javascript converts things.

**Pull request merged**